### DISCUSSION TITLE
Looking into registry alternatives - Docker OSS shutting down

### DISCUSSION THREAD

**Discussion by sigaloid** [spikecodes/libreddit#757] [Ideas]
Today it was announced that Docker would be deleting certain Open Source organizations. https://blog.alexellis.io/docker-is-deleting-open-source-images/

We, too, got this message from Docker. As such we are looking into alternatives: 

- GitHub Content Repository
- Other providers
- Revert to single user (spikecodes/libreddit)

**Comment by sphars** [comment]
Docker came back with an apology and clarified/changed how they're dealing with this: https://www.docker.com/blog/we-apologize-we-did-a-terrible-job-announcing-the-end-of-docker-free-teams/

Whether it's still worth using needs to be evaluated, but certainly have a backup plan just in case.

**Reply by sigaloid** [reply]
That is great news - thank you so much for the link. I'll leave this up in case this ever is relevant in the future, but I'll mark it as solved!

**Reply by onedr0p** [reply]
It would still be nice to support GHCR at least. Dockerhub still has awful rate limits and retention.
